### ISSUE TITLE
Tour's scenes maps

### ISSUE THREAD

**Issue by andrade94** [mpetroff/pannellum#269]
Hello, is there a built-in function for it to display the different scenes that are being used in the tour and not only access it via the hotspots? Like a menu where all the different scenes can be accessed with just a click in their name.

**Comment by mpetroff** [comment]
No, there isn't. Something like this could be build using the API, though.

**Comment by andrade94** [comment]
Really? do you mean with the function addhotspot?

**Comment by mpetroff** [comment]
You could use the `loadScene` method.

**Comment by andrade94** [comment]
Thank you very much, I'll be sure to donate when I get some money.

**Comment by raffaelecolleo** [comment]
Any developments on this matter? My Idea is to build a vector map where I can can see all the divisions of the floor and by clicking on it a can go directly to the scene.

**Comment by gazhay** [comment]
raffaelcolleo,

It's all there now.
I have an svg which contains rects and polygons to represent rooms. I attach click handlers to these and call Viewer.loadScene( sceneID ) and the tour navigates to that room.

No need for any further changes to the source.
